src/documents/views/StartReviewView.js

```diff
@@ -0,0 +1,262 @@
+      />
+    );
+  }
+
+  if (error)
+    return (
+      <Container as={Segment} basic>
+        <Helmet>
+          <title>
+            {`KF Data Tracker - Study reviews - Error ${
+              study ? 'for ' + study.kfId : null
+            }`}
+          </title>
+        </Helmet>
+        <Message
+          negative
+          icon="warning circle"
+          header="Error"
+          content={error.message}
+        />
+      </Container>
+    );
+  return (
+    <Grid as={Segment} basic container columns={1}>
+      <Helmet>
+        <title>
+          {`KF Data Tracker - Study reviews ${
+            study ? 'for ' + study.name : null
+          }`}
+        </title>
+      </Helmet>
+      <Link to={`/study/${kfId}/reviews`} data-testid="back-to-filelist">
+        <Button
+          basic
+          size="mini"
+          labelPosition="left"
+          floated="left"
+          className="mt-6 mb-5"
+        >
+          <Icon name="arrow left" />
+          All Reviews
+        </Button>
+      </Link>
+      <Formik
+        initialValues={{
+          review_title: '',
+          review_desc: '',
+        }}
+        validate={vals => {
+          let errors = {};
+          if (!vals.review_title) {
+            errors.review_title = 'Required';
+          }
+          if (existNames.includes(vals.review_title)) {
+            errors.review_title = 'This name is used.';
+          }
+          if (reviewDesc.length < 3) {
+            errors.review_desc = 'Required';
+          }
+          return errors;
+        }}
+        onSubmit={values => {
+          const rawState = convertToRaw(editorState.getCurrentContent());
+          const mdText = draftToMarkdown(rawState);
+          var submitValue = values;
+          submitValue.review_desc = mdText;
+          createDataReview({
+            variables: {
+              input: {
+                name: values.review_title,
+                description: values.review_desc,
+                study: study.id,
+                versions: [],
+              },
+            },
+          })
+            .then(resp => {
+              const drId = resp.data.createDataReview.dataReview.kfId;
+              history.push(`/study/${kfId}/reviews/${drId}`);
+            })
+            .catch(err => {
+              console.log(err);
+            });
+        }}
+      >
+        {({
+          values,
+          handleChange,
+          handleBlur,
+          handleSubmit,
+          setFieldValue,
+          setFieldTouched,
+          errors,
+          touched,
+          validateForm,
+        }) => (
+          <>
+            <Grid.Row>
+              <Grid.Column width={6}>
+                <h2 className="noMargin">Start a New Data Review</h2>
+                <p>Tell us any additional details about this review</p>
+              </Grid.Column>
+              {files.length > 0 && (
+                <Grid.Column width={10} verticalAlign="bottom">
+                  <Button
```

Comment:
It would be nice if this button was disabled and set to loading state while the mutation is being executed.